app/assets/javascripts/myusa.js

```diff
@@ -146,4 +146,13 @@ $(document).ready(function () {
     });
   });
 
+  $('#application_federal_agency').click(function (e) {
+    var formGroup = $('div.form-group').has('input#application_terms_of_service_accepted');
+    if (e.target.checked) {
+      formGroup.show();
+    } else {
+      formGroup.hide();
+    }
+  });
+
```

Comment:
@polastre ^^^ toggles the TOS checkbox based on whether the federal agency checkbox is set.